### ISSUE TITLE
Multiline command not displaying table correctly when running list

### ISSUE THREAD

**Issue by Plommonsorbet** [BenSchZA/pier#30]
The list command is creating a new table row when dispaying multiline commands, ex:

 Alias                      | tags | Command 
--- | --- | ---
 1-test-single-line         | ` - ` | echo this is a single line command 
 2-test-multi-line          | ` - ` | echo This 
 ` - ` | ` - ` | echo is 
 ` - ` | ` - ` | echo over 
 ` - ` | ` - ` | echo multiple 
 ` - ` | ` - ` | echo lines 
 3-test-another-single-line | ` - ` | echo this is another single line command 

Converted the ascii table to markdown table for nicer viewing. The minus sign can be interpreted as nothing or whitespace.

Config to reproduce error:
```
[scripts.1-test-single-line]
alias = '1-test-single-line'
command = '''
echo this is a single line command
'''

[scripts.2-test-multi-line]
alias = '2-test-multi-line'
command = '''
echo This
echo is
echo over
echo multiple
echo lines
'''

[scripts.3-test-another-single-line]
alias = '3-test-another-multi-line'
command = '''
echo this is another single line command
'''
```

**Comment by Plommonsorbet** [comment]
Perhaps this could be solved by making multiline commands get squished by default and add an option to have full size rows?

**Comment by BenSchZA** [comment]
I think that's a good compromise. If the user wants to see the entire command they can maybe use `list` on that specific alias, or we can add some other flag like `info` or `inspect`.

**Comment by Plommonsorbet** [comment]
This behavior will be fixed in pr #40, so we can close it once that is merged. :)

**Comment by Plommonsorbet** [comment]
resolved.
